packages/wonder-blocks-clickable/src/components/clickable.js

```diff
@@ -211,7 +212,10 @@ export default class Clickable extends React.Component<Props> {
         commonProps: {[string]: any, ...},
     ) => React.Node = (clickableState, commonProps) => {
         const activeHref = this.props.href && !this.props.disabled;
-        const useClient = this.context.router && !this.props.skipClientNav;
+        const useClient =
+            this.context.router &&
+            !this.props.skipClientNav &&
```

Comment:
If I understand correctly... is the idea to remove this `skipClientNav` check after removing all webapp usages?

Reply:
I don't have a task to remove this prop from the API.  There's a couple of uses in webapp where `skipClientNav` is being used in such a way that it can't be removed without making other changes.  See the uses of `skipClientNav` in student-assignment-table.jsx in webapp.